### PR TITLE
Polish layout structure

### PR DESCRIPTION
**Pull request by odincov**
* Homepage layout
* Mobile navigation toggle button
* Min-width 100% for the content

### CODE REVIEW

File: web/app/themes/theme/front-page.php [hunk 1]
```diff
@@ -1,22 +1,34 @@
-<div class="o-wrapper c-home-intro">
-  <div class="u-centering-wrapper">
+<main class="o-wrapper--row c-home-intro u-centering-wrapper" style="background-image: url(<? echo get_template_directory_uri() ?>/assets/images/homepage-intro-image--1.0.jpg);">
```
Comment (rslnk) [inline web/app/themes/theme/front-page.php]: Redundant u-centering-wrapper class

File: web/app/themes/theme/template-timeline.php [hunk 2]
```diff
@@ -3,12 +3,13 @@
  * Template Name: Timeline
  */
 ?>
-<div class="o-wrapper">
+<main class="o-wrapper--row c-home-intro u-centering-wrapper">
```
Comment (rslnk) [inline web/app/themes/theme/template-timeline.php]: Unnecessary class u-centering-wrapper

**Comment by rslnk** [comment]
Nice job! But see my comments.

**Comment by odincov** [comment]
Yep, thanks!

**Comment by rslnk** [comment]
Approved!

**Pull request merged**